src/app/services/xml-parsers/analogue-parser.ts

```diff
@@ -0,0 +1,130 @@
+import { AppConfig } from 'src/app/app.config';
+import { parse, ParserRegister, xmlParser } from '.';
+import { Analogue, AnalogueClass, BibliographicEntry, BibliographicList, GenericElement, Milestone, XMLElement } from '../../models/evt-models';
+import { getOuterHTML } from '../../utils/dom-utils';
+import { AnchorParser, AttributeParser, GenericElemParser, MilestoneParser } from './basic-parsers';
+import { createParser, getID, parseChildren, Parser } from './parser-models';
+import { chainFirstChildTexts, getExternalElements, normalizeSpaces } from 'src/app/utils/xml-utils';
+import { BibliographyParser } from './bilbliography-parsers';
+import { BasicParser } from './quotes-parser';
+
+@xmlParser('evt-analogue-entry-parser', AnalogueParser)
+export class AnalogueParser extends BasicParser implements Parser<XMLElement> {
+
+    elementParser = createParser(GenericElemParser, parse);
+    attributeParser = createParser(AttributeParser, this.genericParse);
+    biblParser = createParser(BibliographyParser, this.genericParse);
+    milestoneParser = createParser(MilestoneParser, this.genericParse);
+    anchorParser = createParser(AnchorParser, this.genericParse);
+
+    analogueMarker = AppConfig.evtSettings.edition.analogueMarkers;
+    biblAttributeToMatch = AppConfig.evtSettings.edition.externalBibliography.biblAttributeToMatch;
+    elemAttributesToMatch = AppConfig.evtSettings.edition.externalBibliography.elementAttributesToMatch;
+    notNiceInText = ['Note', 'BibliographicList', 'BibliographicEntry', 'BibliographicStructEntry',
```

Comment:
Why this name? "notNice"?

Reply:
Those are the usually nested elements (bibl, notes, ref etc...) that will not be displayed in text flow but only in the analogue-component detail view

Reply:
Can you can find a better name?